sdk/appconfiguration/azure-data-appconfiguration/swagger/src/main/java/AppConfigCustomization.java

```diff
@@ -24,6 +26,33 @@ public void customize(LibraryCustomization customization, Logger logger) {
 
         customizeKeyValueFilter(models.getClass("SnapshotSettingFilter"));
         customizeKeyValueFields(models.getClass("SettingFields"));
+        customizeSnapshot(models.getClass("ConfigurationSettingSnapshot"));
+    }
+
+    private void customizeSnapshot(ClassCustomization classCustomization) {
+        classCustomization.getProperty("created").rename("createdAt");
+        classCustomization.getProperty("expires").rename("expiresAt");
+        classCustomization.getProperty("itemsCount").rename("itemCount");
```

Comment:
These could be done with a Swagger transform and adding `x-ms-client` name to the Swagger definition for these properties.